### ISSUE TITLE
Incorrect link

### ISSUE THREAD

**Issue by pr0g** [codereport/Content#6]
Super minor thing but on this page ([Content/blob/main/Talks/2022-12-YouTube/FromCtoCpptoRust/Summary.md](https://github.com/codereport/Content/blob/main/Talks/2022-12-YouTube/FromCtoCpptoRust/Summary.md)), the Godbolt links for `std::accumulate` and `ranges::accumulate` both go to the same URL (https://godbolt.org/z/bfY4qf8Pj).

I believe the `ranges::accumulate` link is incorrect and could be updated.

**Comment by codereport** [comment]
Thanks! Fixed!

**Comment by pr0g** [comment]
> Thanks! Fixed!

No problem, thanks!

On a related topic, I did a little bit of analysis looking at the compile times of the different examples and wrote up the results [here](https://github.com/pr0g/c-to-cpp-range-compile-times).

I found the video and examples you shared really interesting and all being equal, I prefer the code you ended up with, it's just unfortunate there's this subtle cost that can add up pretty significantly across a codebase. I hope in the future more can be done to mitigate these effects (our team has lately been removing 'rangified' code due to the cost it takes to compile).
